action.yml

```
@@ -16,12 +16,12 @@ inputs:
     default: debian:stable-slim
     description: Name of a Docker image to build inside or path of a Dockerfile
     required: true
-  extra-docker-args:
-    description: Additional arguments to 'docker run' of build container
-    required: false
   extra-build-deps:
     description: Extra packages to be installed as build dependencies
     required: false
+  extra-docker-args:
+    description: Additional arguments to 'docker run' of build container
```

Just reword this slightly:
```suggestion
    description: Additional arguments to 'docker run' when starting the build container
```